pkg/apis/nais.io/v1/digdirator_types.go

```diff
@@ -100,6 +100,7 @@ type MaskinportenClient struct {
 
 // MaskinportenClientSpec defines the desired state of MaskinportenClient
 type MaskinportenClientSpec struct {
+	IntegrationName string `json:"integrationName,omitempty"`
```

Comment:
While DigDir does interchangably use `client` and `integration` when referring to the same thing, I believe we should be consistent with their APIs, as well as our own use (i.e. `MaskinportenClient and IDPortenClient`). This should thus be named `ClientName`.

Fields in types should generally follow the official [Go doc comment style guide](https://tip.golang.org/doc/comment) with a descriptive comment, e.g:

```go
type MaskinportenClientSpec struct {
    // ClientName is the client name to be registered at DigDir. It is shown during login for user-centric flows, and is otherwise a human-readable way to differentiate between clients at DigDir's self-service portal.
	ClientName string `json:"clientName,omitempty"`
    ...
```